widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/config/base/AlignmentType.kt

```diff
@@ -0,0 +1,24 @@
+package com.glia.widgets.view.unifiedui.config.base
+
+import android.os.Parcelable
+import android.widget.TextView
+import kotlinx.parcelize.Parcelize
+
+/**
+ * Represents Alignment from remote config.
+ *
+ * @see [com.glia.widgets.view.unifiedui.parse.AlignmentDeserializer]
+ */
+@Parcelize
```

Comment:
It would be great to add such descriptions to all config components, what do you think?

Reply:
I prefer to add javadocs after the feature is complete. Because the implementation, class names, etc can change during development.

Reply:
Agree with @AndriiHorishniiMOC I already changed it a few times.